pygpudrive/env/config.py

```diff
@@ -115,6 +115,10 @@ class EnvConfig:
         gpudrive.numLidarSamples
     )
 
+    #Params to init every object:
```

Comment:
It is not clear to me what the usage would be like, perhaps we can briefly discuss.

Reply:
I changed it to a single init_all_objects boolean for now, since that is what we need for vbd. In the future for wosac, we can easily create separate init modes.